specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/file.json

```diff
@@ -524,10 +544,107 @@
           "minimum": 1,
           "maximum": 5120,
           "description": "The maximum size of the share, in gigabytes. Must be greater than 0, and less than or equal to 5TB (5120)."
+        },
+        "enabledProtocols": {
+          "type": "string",
+          "enum": [
+            "SMB",
+            "NFS"
+          ],
+          "x-ms-enum": {
+            "name": "EnabledProtocols",
+            "modelAsString": true
+          },
+          "description": "Protocols for file shares"
+        },
+        "rootSquash": {
```

Comment:
The enum name is same as one of it's value (both "RootSquash"). This will make the .NET SDK code generated from it build fail.